CHANGES.rst

```diff
@@ -136,6 +136,7 @@ version number.  Breaking changes may occur at any point.
 
   - Processing [GH-793] [GH-812]
   - Racket [GH-799] [GH-873]
+  - Pug -- The pattern has been changed and this is not backwards compatible to Jade. [GH-951] [GH-1084]
```

Comment:
Please move this to the latest version and remove the explanatory sentence.  It's enough for users to see that Jade was removed.